test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_kilometers.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_days.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua
+./test_scripts/Polices/build_options/ATF_P_TC_PTU_Transfer_Several_Apps_Different_HMI_Levels_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_P_PTS_creation_rule_PROPRIETARY.lua
```

Comment:
@dboltovskyi, Additional general checks that can be done for Proprietary flow:
Retry sequence: I didn't manage to find script for checking full retry sequence, but for at least started retry sequence, you can use of the listed:
ATF_PM_change_status_UPDATE_NEEDED_after_timeout_expired.lua
ATF_P_Timeout_wait_response_PTU_PROPRIETARY.lua
ATF_Start_PTU_retry_sequence.lua